src/create_sysroot.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Create a linux sysroot image (tar archive) based on debian
+# This is than used in the build scripts as the --sysroot arguments
+# when compiling linux binaries.
+#
+# See https://xw.is/wiki/Create_Debian_sysroots
+
+set -e
```

Comment:
I prefer using the `-o` flags to set (i.e. `set -o errexit` in this case) since it's much more obvious what they do.

Reply:
Done